### ISSUE TITLE
NTR: HSV present in CSF (Lab: cerebrospinal fluid tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8454]
New term request HSV present in CSF (perlma_100809044446): Viral culture or PCR; herpes simplex virus; cerebrospinal fluid

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000760
- label: Positive CSF herpes simplex virus nucleic acid test
- parents: [Positive CSF nucleic acid pathogen test (HP:0430140)](https://hpo.jax.org/app/browse/term/HP:0430140)
- pmid: 
- def: Detection of herpes simplex virus (HSV) nucleic acid in the cerebrospinal fluid (CSF) by a method such as polymerase chain reaction (PCR) or reverse transcriptase (RT)-PCR.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8454
- synonyms:
